internal/schemamanagement/ts/table_creator.go

```diff
@@ -11,10 +11,12 @@ import (
 )
 
 const (
-	createTableQueryTemplate      = "CREATE TABLE \"%s\"(%s)"
-	columnDefTemplate             = "\"%s\" %s"
-	createHypertableQueryTemplate = "SELECT create_hypertable('%s', '%s');"
-	createTimescaleExtensionQuery = "CREATE EXTENSION IF NOT EXISTS timescaledb"
+	createTableQueryTemplate                = "CREATE TABLE \"%s\"(%s)"
```

Comment:
also discovered another bug, target tables were created as "schema.table" instead of "schema"."table"